Gemfile.lock

```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     middleman-images (0.2.0)
-      middleman-core (>= 4.1.14)
+      middleman-core (>= 4.1.14, < 5.1)
```

Comment:
Where does this change come from?

Reply:
I was wondering about this as well. I just bundled.

Reply:
@pmk1c Maybe this was missing in #61?

Reply:
Yes it was. My mistake. I think this can just stay in here. 👍